examples/emberjs/todomvc/app/routes/active.js

```diff
@@ -0,0 +1,28 @@
+import Route from '@ember/routing/route';
+import { service } from '@ember/service';
+
+export default class ActiveTodos extends Route {
+	@service repo;
+
+	/**
+	 * This should probably be renamed to "data"
+	 * its under active development.
+	 *
+	 * In a real app you'd use this to load your
+	 * _minimally required_ data to show the page.
+	 */
+	model() {
+    let repo = this.repo;
```

Comment:
looks like here is formatting problem

Reply:
everything using tabs now, the diff is outdated :tada: